src/System.Text.Json/tests/Serialization/EnumTests.cs

```diff
@@ -66,5 +134,12 @@ public static void EnumAsUInt64Max()
             SimpleTestClass obj = JsonSerializer.Parse<SimpleTestClass>(s_jsonUInt64EnumMax);
             Assert.Equal(SampleUInt64Enum.Max, obj.MyUInt64Enum);
         }
+
+        [Fact]
+        public static void EnumAsByteMax()
+        {
+            SimpleTestClass obj = JsonSerializer.Parse<SimpleTestClass>(s_jsonByteEnum);
```

Comment:
Can we add tests for serialization as well (i.e. write Enums with out-of-bounds values) on top of deserialization (i.e. what we have now)?

Reply:
- [ ] tests for serialization